server/services/licenceService.ts

```diff
@@ -124,7 +124,7 @@ export default class LicenceService {
       probationTeamDescription: responsibleOfficerDetails.team?.description,
       crn: deliusRecord.otherIds?.crn,
       pnc: deliusRecord.otherIds?.pncNumber,
-      cro: deliusRecord.otherIds?.croNumber,
+      cro: deliusRecord.otherIds?.croNumber || (await this.getCRONumber(prisonerNumber, user)),
```

Comment:
I don't think we need the outer brackets anymore, I think `await this.getCRONumber(prisonerNumber, user)` would suffice